source/release-notes/changelogs/backend/changelog-backend-2022.rst

```diff
@@ -1,3 +1,19 @@
+.. _backend_20220921:
+
+21 September 2022 Release
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Added :method:`mongodb.admin()` and :method:`admin.getDBNames()` in
+  Atlas Functions.
+
+.. _backend_20220907:
+
+07 September 2022 Release
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
i think you want to delete this from the PR, no?

Reply:
Should go away with the rebase